bytelatent/distributed.py

```diff
@@ -463,13 +463,15 @@ def parallelize_model(
         raise ValueError(f"Invalid fsdp_type: {distributed_args.fsdp_type}")
 
     if distributed_args.selective_activation_checkpointing:
-        model = checkpoint_wrapper(
-            model,
-            context_fn=partial(
-                create_selective_checkpoint_contexts,
-                get_default_policy(no_recompute_ops),
-            ),
-        )
+        for module in [model.global_transformer, model.local_encoder, model.local_decoder]:
```

Comment:
Maybe just leave a comment that this only works for BLT, but that entropy model doesn't need it.